lib/mergeusersearch.php

```diff
@@ -118,19 +130,20 @@ public function search_users($input, $searchfield){
                 );
 
                 $sql =
-                   'SELECT *
+                   "SELECT *
                     FROM {user}
                     WHERE
-                        id = :userid OR
+                        ($whereid OR
```

Comment:
Revisit according to the above changes, if necessary.

Reply:
This is now `'id = :userid OR` again.